frontend/src/components/pages/VolunteerDashboard/index.tsx

```diff
@@ -0,0 +1,33 @@
+import { Container } from "@chakra-ui/react";
+import React, { useContext, useEffect, useState } from "react";
+
+import VolunteerAPIClient from "../../../APIClients/VolunteerAPIClient";
+import AuthContext from "../../../contexts/AuthContext";
+import { Status } from "../../../types/AuthTypes";
+import PendingPage from "./PendingPage";
+import ScheduledVolunteerShiftsPage from "./ShiftsPage";
+
+const VolunteerDashboard = () => {
+  const [volunteerStatus, setVolunteerStatus] = useState<Status>();
+  const { authenticatedUser } = useContext(AuthContext);
+
+  const getVolunteerData = async () => {
+    const volunteerResponse = await VolunteerAPIClient.getVolunteerByUserId(
+      authenticatedUser!.id,
+    );
+    setVolunteerStatus(volunteerResponse.status);
+  };
+
+  useEffect(() => {
+    getVolunteerData();
+  }, []);
```

Comment:
we shouldn't need any of this to make an API call, the volunteer information is already stored when they are logged in through the context: 
https://github.com/uwblueprint/community-fridge-kw/blob/3fc83aa50c2bce8913ff25403513f1606c0e4f88/frontend/src/contexts/VolunteerContext.ts#L8
you can reference it like 
```typescript
const { volunteerStatus } = useContext(VolunteerContext)
```